src/frontend/utils/telemetry/Instrumentation.js

```diff
@@ -17,9 +17,17 @@ const {envDetector, hostDetector, osDetector, processDetector} = require('@opent
 //   opentelemetry.DiagLogLevel.INFO
 // );
 
+console.log("Instrumenting node with otel");
 const sdk = new otelsdk.NodeSDK({
-  traceExporter: new OTLPTraceExporter(),
-//  logRecordProcessor: new otelsdk.logs.BatchLogRecordProcessor(new OTLPLogExporter()),
+  // n.b. - the service for the next.js backend is being
+  // sent to Honeycomb as 'api-gateway' - this is done
+  // in our collector. Don't bother to set the serviceName
+  // here!
+
+  traceExporter: new OTLPTraceExporter({
+    url: '/otlp-http/v1/traces'
```

Comment:
Not massively important, but we could perhaps change the envoy config to match on `/v1/` instead of `/otlp-http`. If we're asking someone to deploy this to production, I'd probably have the LB do that, instead of trying to do a path rewrite.

It's not something that would block anything, just something that's interesting in terms of how we frame the backend deployment of the pipeline for production configuration